posthog/api/test/test_action.py

```diff
@@ -44,6 +30,25 @@ def test_create_and_update_action(self, patch_delay):
         self.assertEqual(action.steps.get().selector, "div > button")
         self.assertEqual(response["steps"][0]["text"], "sign up")
 
+        # Assert analytics are sent
```

Comment:
Nitpick comment, feel free to ignore: how hard would it be to separate this test out?

Ideally tests should only 'test' a single thing - mixing multiple concerns gets messy fast.